web_scraper/README.md

```diff
@@ -0,0 +1,23 @@
+# Webscraper
+
+The module has been built using [scrapy](https://docs.scrapy.org/en/latest/intro/overview.html#scrapy-at-a-glance), an application framework used for crawling web sites and extracting structured data.
+
+As of now, the WebScraper has just 1 spider that is used to crawl the CVE URLs. 
+
+## Setup:
+Installing [scrapy](https://docs.scrapy.org/en/latest/intro/install.html).
+
+## Running the Code:
+To run the code, following steps can be taken:
+
+**Note: all the commands given below should be run from this directory i.e. `~/deep-learning-containers/web_scraper` itself.**
+
+1. Set the following environment variable:
+`export SCRAPE_URL_LIST="https://ubuntu.com/security/CVE-2016-1585 https://ubuntu.com/security/CVE-2021-29973"`. <br> The `SCRAPE_URL_LIST` environment variable consists of the list of space seperated CVE URLs to scrape. 
+
+2. In the deep-learning-containers root folder, remove the file scraped_data using the following command `rm -r ../scraped_data.json`.
+
+3. Start the crawl by using the following command `scrapy crawl cve -o ../scraped_data.json`
```

Comment:
is it possible to send input to the spider from the command line interface rather than using environment?

Reply:
There is a way to pass arguments. I am trying to figure out a way to directly call the crawler class from within the script. That would take away the need of all the command line operations.